lib/src/backend/invoker.dart

```diff
@@ -289,6 +289,11 @@ class Invoker {
     _controller.addError(error, stackTrace);
     removeAllOutstandingCallbacks();
 
+    if (!liveTest.test.metadata.chainStackTraces) {
+      _printsOnFailure.add("Consider enabling the flag chain-stack-traces to "
```

Comment:
Could we make this a little more specific, perhaps:

"Consider ...\n For example, pub run test --chain-stack-traces"